### PR TITLE
optimisation: extract common M861 code

### PR DESCRIPTION
**Pull request by gudnimg**
Change in memory:
Flash: -162 bytes
SRAM: 0 bytes

### CODE REVIEW

File: Firmware/Marlin_main.cpp [hunk 1]
```diff
@@ -3800,6 +3800,26 @@ static void gcode_G92()
...
+static void gcode_M861_print_pinda_cal_eeprom() {
+    int16_t usteps;
+    for (uint8_t i = 0; i < 6; i++) {
+        usteps = 0;
+        if(i > 0) {
+            usteps = eeprom_read_word((uint16_t*) EEPROM_PROBE_TEMP_SHIFT + (i - 1));
+        }
+        float mm = ((float)usteps) / cs.axis_steps_per_unit[Z_AXIS];
+        i == 0 ? SERIAL_PROTOCOLPGM("n/a") : SERIAL_PROTOCOL(i - 1);
+        SERIAL_PROTOCOLPGM(", ");
```
Comment (DRracer) [inline Firmware/Marlin_main.cpp]: Please check, that gcc is able to merge the duplicate `", "` strings.

Comment (gudnimg) [reply Firmware/Marlin_main.cpp]: @DRracer do you mean by adding `", "` to `messages.cpp`?

Comment (DRracer) [reply Firmware/Marlin_main.cpp]: Even by extracting to a `static const char something[] PROGMEM;` locally ...

Comment (DRracer) [reply Firmware/Marlin_main.cpp]: ... and yes, having a collection of these tiny strings at one spot and reusing them across the whole firmware might actually bring some savings.

Comment (gudnimg) [reply Firmware/Marlin_main.cpp]: Done ✔️

**Review by DRracer — APPROVED** [review]
looks ok to me :+1:

**Pull request merged**